src/version.h

```diff
@@ -24,9 +24,15 @@
   #define KIWIX_TOOLS_VERSION "undefined"
 #endif
 
+#include <kiwix/version.h>
+#include <zim/version.h>
+
 void version()
 {
-  std::cout << KIWIX_TOOLS_VERSION << std::endl;
+  std::cout << KIWIX_TOOLS_VERSION << std::endl << std::endl;
+  kiwix::printVersions();
+  std::cout << std::endl;
+  zim::printVersions();
```

Comment:
Some libs will be duplicated between `kiwix::printVersions()` and `zim::printVersions()`. Maybe it makes sense to print a line before each of them telling what follows. Or the `printVersions()` functions could produce a tree-like output:

```
libkiwix 10.1.2
+ libzim 7.1.0
+ libicu 7.6

libzim 7.1.0
+ libzstd 1.2.3
+ libxapian 4.5.6
+ libicu 7.8.9
```

Reply:
I agree, I have implemented this readability improvement in kiwix/libkiwix#665.

I have as well created a `getVersionsString()`, this is was not mandatory here, but I need such a function with binaries using Docopt (in particular for a few command line tools in openzim/zim-tools).

If you agree with this two improvements, I plan to make the same i new PR for openzim/libzim.